### DISCUSSION TITLE
Supabase CLI with self-hosted and db diff

### DISCUSSION THREAD

**Discussion by captainR0bbo** [supabase/supabase#11564] [Questions]
We used to be able to use the db diff tool with a self-hosted instance. It stored the instance connection in supabase/.temp/remote-db-url.

With the latest CLI, it only lets you link to a supabase.com project. I see no way to use it with a self-hosted instance. 

Is it possible?

**Comment by SteveChurch** [comment]
I have also been wondering the same @captainR0bbo - Also, probably for a separate thread but how do we deploy functions to self-hosted?
